README.md

```diff
@@ -12,9 +12,27 @@ python manage.py crawl_course
 python manage.py update_index
 ```
 
-For auto decaptcha feature, it requires ``tesseract 3.03``
-
 To clear all contents in db, use ``python manage.py crawl_course clear``
 
 To update all syllabus, use ``python manage.py update_syllabus``
 
+For auto decaptcha feature, it requires ``tesseract 3.03``
+
+# Install dependency
+- tesseract
+    - mac: `brew install tesseract --devel # Without --devel, it will install tesseract 3.02`
+    - ubuntu: `sudo apt-get install tesseract-ocr`
+
+
+# Setting
+We use mysql as database. The default setting will find a config file in 'NTHU_Course/mysql.ini'. The following is a sample config file:
```

Comment:
Can you create a script which will prompt user to type in these info if the file does not exist?